napari/_qt/qt_event_loop.py

```diff
@@ -61,7 +61,59 @@ def set_app_id(app_id):
 _IPYTHON_WAS_HERE_FIRST = 'IPython' in sys.modules
 
 
-def get_app(
+def get_app(*args, **kwargs) -> QApplication:
+    """Get or create the Qt QApplication.
```

Comment:
I feel like we can just delete this docstring and just say, deprecated, use `get_qapp`.

Maybe we could add a todo to say delete in napari 0.6.0 (I assume next major release dep is fine?) Also lets add the todo to the other `get_app` as well!